### DISCUSSION TITLE
@mantine/dev TimeInput Component supports milliseconds

### DISCUSSION THREAD

**Discussion by jiananshi** [mantinedev/mantine#3888] [Feature requests]
hello mantine devs, In my project I need not only seconds but also milliseconds, I wonder if this feature is already in roadmap?

**Comment by rtivital** [comment]
TimeInput component is built with `input[type="time"]`, it does not support any extra features, you can find `input[type="time"]` docs here – https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time
